### PR TITLE
Static execute dir name

### PR DESCRIPTION
**Pull request by majamassarini**
Merge before packit/packit#2081 and packit/packit.dev#743

Fixes packit/packit-service#1956

### CODE REVIEW

**Comment by majamassarini** [comment]
recheck

**Comment by majamassarini** [comment]
recheck

File: sandcastle/utils.py [hunk 1]
```diff
@@ -97,11 +97,14 @@ def clean_string(s: str) -> str:
     '[a-z0-9]( [-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*'
 
     'must be no more than 63 characters'
+
+    remove 21 characters for make room to the timestamp
+    remove 5 characters for make room to the - {{ deployment }} -
```
Comment (nforro) [inline sandcastle/utils.py:102]: Could you please clarify why is this needed?

Comment (majamassarini) [reply sandcastle/utils.py]: I rewrote it in a different way, let me know if now it is clear.

Comment (nforro) [reply sandcastle/utils.py]: That's better, thanks, but it still isn't clear to me why there is now a need to make a room for a timestamp (that will be appended to the pod name I assume) and it wasn't before.

Comment (nforro) [reply sandcastle/utils.py]: I mean, was it a bug?

**Review by nforro — APPROVED** [review]
